package.json

```diff
@@ -49,8 +49,8 @@
     "nx": "^16.9.1",
     "sinon": "^15.1.0",
     "typescript": "^4.8.3",
-    "vite": "^4.4.9",
-    "web-component-analyzer": "2.0.0-next.4",
+    "vite": "^5.0.10",
+    "web-component-analyzer": "2.0.0",
```

Comment:
Is there any reason to this version update ? Please check with Mos, this may potentially introduce bug in API generation.

Reply:
It has a security fix in the package, I think it should compatible with 2.0.0. @Sarin-Udompanish  Can you help me check it?

Reply:
LGTM
![Uploading Screenshot 2566-12-21 at 16.35.28.png…]()